woodwork/tests/accessor/test_statistics.py

```diff
@@ -603,17 +603,28 @@ def test_describe_with_include(sample_df):
     assert len(semantic_tags_df.columns) == 2
 
     logical_types_df = sample_df.ww.describe([Datetime, BooleanNullable])
-    assert 'signup_date', 'is_registered' in logical_types_df.columns
-    assert len(logical_types_df.columns) == 2
+    for c in ['signup_date', 'is_registered', 'datetime_with_NaT']:
+        assert c in logical_types_df.columns
+    assert len(logical_types_df.columns) == 3
 
     multi_params_df = sample_df.ww.describe(['age', 'tag1', Datetime])
-    expected = ['full_name', 'age', 'signup_date']
+    expected = ['full_name', 'age', 'signup_date', 'datetime_with_NaT']
     for col_name in expected:
         assert col_name in multi_params_df.columns
     multi_params_df['full_name'].equals(col_name_df['full_name'])
     multi_params_df['full_name'].equals(sample_df.ww.describe()['full_name'])
 
 
+def test_pandas_nullable_integer_quantile_fix():
+    """Should fail when https://github.com/pandas-dev/pandas/issues/42626 gets fixed"""
+    if pd.__version__ != '1.3.0':
```

Comment:
I assume this is just to get a test failure that prompts us to implement changes once this bug is fixed?

If so, and we leave this test in, maybe we should mark this `if` block with a `# pragma: no cover` so it doesn't cause coverage issues.